### PR TITLE
Convert to plugin2.

### PR DESCRIPTION
**Pull request by SysPete**
Next Dancer2 release includes a major overhaul to Plugin (known as plugin2) which breaks this plugin. Attempts have been made to prevent breakage of existing plugins but this is not possible in all cases (like this one) so a new release of this plugin will be needed as soon as next Dancer2 release occurs.

If you would like me to handle the next release of this plugin in sync with Dancer2 please let me have co-maint and I will sort. I have a handful of other plugins to release at the same time so one more is no big deal.

### CODE REVIEW

File: lib/Dancer2/Plugin/Locale.pm [hunk 1]
```diff
@@ -18,21 +18,23 @@ use File::Spec;
...
+
+sub locale {
     my $dsl = shift;
 
     if (@_) {
         return Dancer2::Plugin::Locale::Obj->get_handle( grep( { defined } (@_) ), 'en' );    # multiton already via Locale::Maketext::Utils
     }
 
-    my $conf = plugin_setting();
+    my $conf = $dsl->config;
```
Comment (drmuey) [inline lib/Dancer2/Plugin/Locale.pm:30]: This variable isn't used so it could be dropped no?

**Comment by drmuey** [comment]
Thanks @SysPete! Is there an ETA on the Dancer2 plugin update?

File: lib/Dancer2/Plugin/Locale.pm (hunk 1 above)
Comment (SysPete) [inline lib/Dancer2/Plugin/Locale.pm:30]: Yes - I'll update soon.

**Comment by SysPete** [comment]
I'm not sure about ETA except that I expect it to be fairly soon.

File: lib/Dancer2/Plugin/Locale.pm (hunk 1 above)
Comment (drmuey) [inline lib/Dancer2/Plugin/Locale.pm:30]: cool, thx!

**Pull request merged**